frontend/src/app/app-routing.module.ts

```diff
@@ -3,10 +3,20 @@ import {RouterModule, Routes} from '@angular/router';
 import {NotFoundComponent} from './not-found.component';
 
 
-const appRoutes: Routes = [{
-  path: 'applicationDashboard',
-  loadChildren: './application-dashboard/application-dashboard.module#ApplicationDashboardModule'
-},
+const appRoutes: Routes = [
+  {
+    path: 'applicationDashboard',
+    loadChildren: './modules/application-dashboard/application-dashboard.module#ApplicationDashboardModule'
+  },
+  {
+    path: 'landing',
+    loadChildren: './modules/landing/landing.module#LandingModule'
+  },
+  {
+    path: '',
+    loadChildren: './modules/landing/landing.module#LandingModule',
+    pathMatch: 'full'
+  },
```

Comment:
The landing page needs to be reachable at for example `https://openspeedmonitor.org/` and  `https://openspeedmonitor.org/landing,  `https://openspeedmonitor.org/landing/continueSetup`. The first case is a special case that *only* fits `''` (nothing behind it, therefore `pathjMatch: 'full'`), otherwise all routing would be handled by the routing of the landing module. This way only secondlevel routing (like `/landing/continueSetup`) is handled by the module.